docs/manage/users-roles/roles/role-capabilities.md

```diff
@@ -4,258 +4,319 @@ title: Role Capabilities
-| Triage | Edit | Create, edit,and delete triage events. |
-| Triage | Bulk physical delete | Perform bulk deletion of triage events. |
-| Folders | Edit | Create, edit, and delete folders. |
-| Attachments | Access | Access all [attachments](/docs/cloud-soar/incidents-triage/#documentation-tab). |
-| Attachments | Edit | Create, edit, and delete attachments. |
-| Incident Playbook | Access | Access all [incident playbooks](/docs/cloud-soar/incidents-triage/#playbooks). |
-| Incident Playbook | Edit | Create, edit, and delete incident playbooks. |
-| Incident Playbook | Manage | Manage incident playbooks. |
-| Note | Access | Access all [notes](/docs/cloud-soar/incidents-triage/#notes). |
-| Note | Edit | Create, edit, and delete notes. |
-| War Room | Use | Be able to use the [War Room](/docs/cloud-soar/incidents-triage/#war-room).  |
-| Settings General | Configure | Configure [settings](/docs/cloud-soar/settings/). |
-| User Management | Groups | Manage [groups](/docs/cloud-soar/settings/#groups). |
-| Notification | Configure | Configure [notifications](/docs/cloud-soar/settings/#notifications). |
-| Customization | Logo | Customize the logo. |
-| Customization | Fields | Customize [fields](/docs/cloud-soar/settings/#custom-fields). |
-| Customization | Incident Labels | Customize incident labels. |
-| Customization | Triage | Customize triage. |
-| Audit and Information | License Information | View license [audit and information](/docs/cloud-soar/legacy/legacy-cloud-soar-global-functions-menu/#audit-and-information). |
-| Audit and Information | Audit Trail | View audit trail information. |
-| Audit and Information | Configure Audit Trail | Configure audit trail information. |
-| API | Use | Use [APIs](/docs/api/cloud-soar/). |
-| API | Api Admin | Have admin access to APIs. |
-| API | Email Read | Read emails. |
-| API | Email Edit | Create, edit, and delete emails. |
-| Incident Templates | Access | Access all [incident templates](/docs/cloud-soar/automation/#incident-templates). |
-| Incident Templates | Configure | Configure templates. |
-| Automation Rules | Access | Access [automation rules](/docs/cloud-soar/automation/#automation-rules). |
-| Automation Rules | Configure | Configure automation rules. |
-| Entities | Access | Access all [entities](/docs/cloud-soar/incidents-triage/#entities). |
-| Entities | Manage | Manage entities. |
-| Entities | Bulk Physical Delete | Perform bulk deletion of entities. |
-| Report | Access | Access your [reports](/docs/cloud-soar/incidents-triage/#report). |
-| Report | Access all | Access all reports. |
-| Dashboard | Access | Access your [dashboards](/docs/cloud-soar/incidents-triage/#dashboards). |
-| Dashboard | Access all | Access all dashboards. |
-| Widgets | Use all | Use all [widgets](/docs/cloud-soar/incidents-triage/#create-widgets). |
-
-### Legacy Cloud SOAR role capabilities
+| View Unmasked Data | ? | ? |
+-->
+
+## Organizations
```

Comment:
I think these are only valid in the UI. I couldn't find them in the API doc. If that's the case, I'd change the `<ApiCreateRole/>` column header and maybe add a note above it